extensions/ql-vscode/src/query-history.ts

```diff
@@ -59,16 +59,15 @@ interface QueryHistoryDataProvider extends vscode.TreeDataProvider<CompletedQuer
 /**
  * Tree data provider for the query history view.
  */
-class HistoryTreeDataProvider implements QueryHistoryDataProvider {
+class HistoryTreeDataProvider extends DisposableObject implements QueryHistoryDataProvider {
   /**
    * XXX: This idiom for how to get a `.fire()`-able event emitter was
```

Comment:
Get rid of this comment. It is a fairly standard idiom throughout all of vscode.

Reply:
Still planning to do this?

Reply:
done